lib/locations_map.rb

```diff
@@ -64,6 +64,11 @@ def load_home_locations
               col_sep: "\t").each_with_object({}) do |(home_location, library_code, folio_code), hash|
       library_code = { 'LANE' => 'LANE-MED' }.fetch(library_code, library_code)
 
+      # Handle mapping of MEDIA-CRES to MEDIA-CENTER, which was MEDIA-MTXT in Symphony
+      # https://github.com/sul-dlss/folio_migration/blob/63166e3163bcee0562dd0c9b96d313e1adf968ac/mapping_files/temp_locations.tsv#L19
+      # https://stanfordlib.slack.com/archives/G018TJ20XGE/p1695672377659539
+      library_code = 'MEDIA-MTXT' if library_code == 'MEDIA-CENTER'
```

Comment:
Is it possible to integrate this into the literal hash on line 65?

Reply:
updated